src/datasets/utils/py_utils.py

```diff
@@ -146,21 +147,21 @@ def __get__(self, obj, objtype=None):
         return self.fget.__get__(None, objtype)()
 
 
-class memoized_property(property):  # pylint: disable=invalid-name
-    """Descriptor that mimics @property but caches output in member variable."""
-
-    def __get__(self, obj, objtype=None):
-        # See https://docs.python.org/3/howto/descriptor.html#properties
-        if obj is None:
-            return self
-        if self.fget is None:
-            raise AttributeError("unreadable attribute")
-        attr = "__cached_" + self.fget.__name__
-        cached = getattr(obj, attr, None)
-        if cached is None:
-            cached = self.fget(obj)
-            setattr(obj, attr, cached)
-        return cached
+# class memoized_property(property):  # pylint: disable=invalid-name
```

Comment:
I just noticed that you also commented these.
Can you uncomment them ?
We can discuss removing them in another PR

Reply:
Done @lhoestq

Reply:
Thanks !